internal/connector/connector.go

```diff
@@ -31,10 +32,12 @@ import (
 )
 
 type Options struct {
-	Server ServerOptions
-	Name   string
-	CACert string
-	CAKey  string
+	Server        ServerOptions
+	Name          string
+	CACert        string // TODO: types.StringOrFile
```

Comment:
While just a comment, I absolutely love the idea of `types.StringOrFile`

Reply:
Done in #3220 !